src/capi/azext_capi/custom.py

```diff
@@ -623,6 +624,11 @@ def create_workload_cluster(  # pylint: disable=too-many-arguments,too-many-loca
     with Spinner(cmd, "Waiting for workload cluster nodes to be ready", "✓ Workload cluster is ready"):
         kubectl_helpers.wait_for_nodes(workload_cfg)
 
+    if wait_for_nodes:
+        total_machine_count = int(control_plane_machine_count) + int(node_machine_count)
+        with Spinner(cmd, "Waiting for all workload cluster nodes to be ready", "✓ Workload cluster is ready"):
```

Comment:
I wonder if we could get the Spinner to update with the count of nodes that are Ready, like:
`Waiting for workload cluster nodes to be ready: 1/5`

That might require subclassing `Spinner`... Not essential, maybe we could create an issue for doing that as a later improvement.